src/lib/Layout/SideBarLayout/SideBarLayoutContext.tsx

```diff
@@ -1,34 +0,0 @@
-import { createContext, PropsWithChildren, ReactNode, useContext } from "react";
-
-interface SideBarLayoutProviderProps {
-  theme?: "dark" | "light";
-  footer?: ReactNode;
-  brand?: ReactNode;
-  userDropDownMenuToggle?: ReactNode;
-  userDropDownMenu?: ReactNode;
-}
-
-interface SideBarLayoutContextProps extends SideBarLayoutProviderProps {
-  theme: "dark" | "light";
-}
-
-const SideBarLayoutContext = createContext<SideBarLayoutContextProps | null>(null);
-
-const SideBarLayoutProvider = (props: PropsWithChildren<SideBarLayoutProviderProps>) => {
```

Comment:
@severinoAmmirati @manni497 i wanted to test it with whitelabel, there the SideBarLayoutProvider is used and that is not exposed anymore... how does this has to be migrated? and i think this would also need to be mentioned in the changelog.

Reply:
@neotob It was merged with PanelSideBarProvider as they are the same. I will mention it as breaking changes in Changelog

Reply:
@neotob I would avoid the breaking change on the old SideBar.. As we thought this is used. So I would revert the file, making sure this is not used in the new menu. @neotob What do you think?

Reply:
@neotob @manni497 Also for me revert would be fine.

Reply:
i think a "small" breaking change where you only have to change the name would be okay to prevent having duplicated code 🤔 would it be possible to make it at least 100% compatible to the old one? currently its complaining about following stuff:
![image](https://github.com/neolution-ch/react-pattern-ui/assets/103021062/17b0b215-5eaa-465f-8570-0d529fca94a9)
in whitelabel the children are sent as "real" children:
![image](https://github.com/neolution-ch/react-pattern-ui/assets/103021062/f3ce9c37-f59f-4648-8168-326ae80cfaab)

Reply:
@neotob As discussed, old context rolled back

Reply:
tested on whitelabel and works 👍